README.md

```diff
@@ -1,20 +1,275 @@
-# fullstory-data-layer-observer
+# FullStory Data Layer Observer
 
 [![CircleCI](https://circleci.com/gh/fullstorydev/fullstory-data-layer-observer.svg?style=svg)](https://circleci.com/gh/fullstorydev/fullstory-data-layer-observer)
 
-## Note: This API is currently in private beta.
+FullStory Data Layer Observer (DLO) is a small but powerful JavaScript utility that makes integrating analytics data easier.  Through a flexible syntax and rules-driven approach, you can reference data from a data layer, perform intermediate changes, and send the result to FullStory - all without writing any custom JavaScript.
```

Comment:
One thing that crystalized in my head after reading through the docs so far was how closely we match the classic ETL pattern. ETL rules are constructed by the combination of `source` fields selecting properties from a JSON object, `operator` chains that transform the structure and values of this object, and `destination` properties that load the transformed data into FullStory.

Would it be a good idea to outline this connection between ETL and the way DLO rules are built explicitly in the intro?

Reply:
"DLO is ETL for JSON" 

:)

Reply:
Agreed; I think we keep that in the back of our minds as we revise the overall messaging.  We may find ourselves repositioning the README quite a bit in the next month or so.